src/index.js

```diff
@@ -1,9 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './styles/index.scss';
-import { App } from './router/App';
+import App from './router/App';
+import { createStore, applyMiddleware, compose } from 'redux';
+import rootReducer from './reducers/store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+
+const store = createStore(
+  rootReducer,
+  compose(
+    applyMiddleware(thunk),
```

Comment:
Are you using `thunk` right now or you just anticipate using it?

Reply:
Just unticipate using it.